src/fmcalc/fmcalc.cpp

```diff
@@ -1066,11 +1070,15 @@ void fmcalc::init_itemtotiv()
 			exit(-1);
 		}
 		last_item_id = itm.id;
-		item_to_tiv_[itm.id] = coverages[itm.coverage_id];
+		if (!used_tiv[itm.coverage_id]) {
+			item_to_tiv_[itm.id] = coverages[itm.coverage_id];
+			used_tiv[itm.coverage_id] = true;
+		}
+		else {
+			item_to_tiv_[itm.id] = 0;
+		}
 		i = fread(&itm, sizeof(itm), 1, fin);
 	}
-	fclose(fin);
```

Comment:
I think we still need to close the file

Reply:
i haven't changed this bit. what is your suggestion?